content/tutorials/dsx-subscribe.textile

```diff
@@ -0,0 +1,249 @@
+
+```[html]
+<html>
+<head>
+  <title>DSX subscriber tutorial</title>
+  <script src="https://cdn.ably.io/lib/ably.min-1.js" crossorigin="anonymous"></script>
+  <script
+  src="https://code.jquery.com/jquery-3.3.1.min.js"
+  integrity="sha256-FgpCb/KJQlLNfOu91ta32o/NMZxltwRo8QtmkMRdAu8="
+  crossorigin="anonymous"></script>
+</head>
+<body>
+  <div>
+    <div>
+      Station:
+      <select id="naptan-code">
+        <option value="northern:940GZZLUEUS">Euston Underground Station</option>
+        <option value="northern:940GZZLUKSX">Kings Cross Underground Station</option>
+        <option value="jubilee:940GZZLUWSM">Westminster Underground Station</option>
+        <option value="northern:940GZZLUWLO">Waterloo Underground Station</option>
+      </select>
+    </div>
+  </div>
+  <table id="table-tfl" class="c-flashbox" style="margin-top: 20px;">
+    <thead>
+      <tr>
+        <th>Due</th>
+        <th>Destination</th>
+        <th>Platform</th>
+        <th>Line</th>
+      </tr>
+    </thead>
+    <tbody>
+    </tbody>
+  </table>
+<script src="main.js"></script>
+</body>
+</html>
+```
+
+h2. Step 4 - Determining the channel names
+
+In DSX, the channels available in any product are be determined by the naming convention described below:
+
+[product:<strong>DSX-ID</strong>]:<strong>NAMESPACE</strong>:<strong>OPTIONAL-CATEGORY</strong>
+
+For the current product, we have three categories of channels, each containing a different set of data:
+
+1. Arrivals at a station, with namespace @tube:{{line}}:{{station}}:arrivals@
+2. Disruptions on a line, with namespace @tube:{{line}}:disruptions@
+3. Disruptions on all lines, with namespace @tube:disruptions@
+
+h2. Step 5 - Attaching and subscribing to a channel
+
+Let's go ahead and attach our client to the arrivals channel, i.e. the first one from the list above. If you are already a user of "Ably's DSN platform":/realtime/channels, you'll observe that the subscription is established in the same way in DSX, as in DSN.
+
+First, let's instance the Ably client as shown below. Make sure to replace the placeholder with your actual API key.
+
+```[javascript]
+var apiKey = 'YOUR-API-KEY';
+var ably = new Ably.Realtime(key: apiKey);
+```
+
+Note that in this tutorial we've used the API key directly to authenticate with Ably. This is generally "not a recommended authentication strategy":/best-practice-guide#using-api-key-client-side as the API key is prone to compromise while traveling on the internet. Ably offers "Token Authentication":/core-features/authentication#token-authentication which is more secure and is generally used in production environments. 
```

Comment:
I think the issue is less the possibility of compromise whilst authenticating (we use end to end encryption and all that), rather than it means in any clients which use this code they'll have unrestrictable access to the product, and can be distributed and all that.